### DISCUSSION TITLE
Cannot create Azure container instance using the docker.io image

### DISCUSSION THREAD

**Discussion by daveyq1** [mockoon/mockoon#1438] [Help] [answered]
Our Azure DevOps yaml pipeline creates a container instance using the docker.io image. This last worked fine on 1st July, but as of today the following error is being thrown `ERROR: (RegistryErrorResponse) An error response is received from the docker registry 'docker.io'. Please retry later.
Code: RegistryErrorResponse
Message: An error response is received from the docker registry 'docker.io'. Please retry later.
`
This is the AzureCli task which is used to create the container
`- task: AzureCLI@2
  name: CreateContainer
  displayName: Create container
  inputs:
	azureSubscription: ${{ parameters.subscription }}
	scriptType: bash
	scriptLocation: inlineScript
	inlineScript: |
	  az container create \
		--resource-group '${{ parameters.resourceGroup }}' \
		--name '${{ parameters.name }}' \
		--image 'docker.io/mockoon/cli:latest' \
		--ports ${{ parameters.port }} \
		--location '${{ parameters.location }}' \
		--dns-name-label '${{ parameters.name }}' \
		--command-line "mockoon-cli start --data '$(publicUrl)' --port ${{ parameters.port }}"`
I've tried using a specific version of the mockoon image e.g. 8.2.0 but get the same error

**Comment by 255kb** [comment, marked as answer]
This issue seems to be an issue on Docker's side, or a connectivity problem between Azure DO and Docker.io. 

I'm not aware of anything affecting our image hosted on Docker hub.

**Reply by daveyq1** [reply]
Thanks @255kb this has been posted on https://github.com/Azure/azure-cli/issues/29300
